m0t1fs/linux_kernel/st/m0t1fs_rpc_cancel_test.sh

```diff
@@ -291,28 +291,28 @@ rcancel_cancel_during_write_test()
 	# troubleshooting, in case required.
 	echo "Test: ls for $wt_write_nr_files files before restore (Succeeds if 'Operation canceled' is not received. Shall succeed for all, if rcancel_md_redundancy > 1)"
 	for ((i=0; i<$wt_write_nr_files; ++i)); do
-		ls -l $wt_write_file_base$i
+		ls -l "$wt_write_file_base""$i"
 		echo "ls_rc: $?"
 	done
 
 	# Verify that some dd operations were indeed canceled.
 	# It is to verify that some RPC items were indeed canceled through
 	# RPC session cancelation. Some items are going to get canceled
 	# while attempting to be posted after session cancelation.
-	num=`grep -n "dd: " $MOTR_TEST_LOGFILE | grep "writing" | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
+	num=`grep -n "dd: " "$MOTR_TEST_LOGFILE" | grep "writing" | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
 	echo "dd write processes canceled : $num"
-	if [ $num -eq 0 ]; then
+	if [ "$num" -eq 0 ]; then
 		echo "Failed: No dd writing operation was canceled"
 		unmount_and_clean
 		return 1
 	fi
 
-	num=`grep -n "dd: closing" $MOTR_TEST_LOGFILE | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
+	num=`grep -n "dd: closing" "$MOTR_TEST_LOGFILE" | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
 	echo "dd closing processes canceled : $num"
 
-	num=`grep -n "ls: cannot access" $MOTR_TEST_LOGFILE | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
+	num=`grep -n "ls: cannot access" "$MOTR_TEST_LOGFILE" | grep "Operation canceled" | grep "$wt_write_file_base" | wc -l | cut -f1 -d' '`
 	echo "ls processes canceled : $num"
-	if [ $num -gt 0 ] && [ $rcancel_md_redundancy > 1 ]; then
+	if [ "$num" -gt 0 ] && [ $rcancel_md_redundancy > 1 ]; then
```

Comment:
If we do double quote for `$num`, why not for `$rcancel_md_redundancy`?

Reply:
done